packages/extension-ui/src/components/SplashHandler.tsx

```diff
@@ -73,10 +75,16 @@ export default styled(SplashHandler)`
   flex-direction: column;
   height: 100%;
 
-  > *:not(.splash):not(.header) {
+  > *:not(.splash):not(.header):not(${ScrollWrapper}):not(${Steps}) {
     padding-left: 16px;
     padding-right: 16px;
   }
+  
+  > *:not(.splash):not(.header)${ScrollWrapper} {
+    scrollbar-gutter: stable both-edges;
+    padding-left: 8px;
+    padding-right: 8px;
```

Comment:
The scrollbar gutter adds padding on its own, so I had to lower the padding we're setting explicitly to account for the gutter. Unforutanatelly (and obviously) the gutter has different sizes between browsers, so the visual gross paddings will differ, but I think this is how it has to be.